Package/Assets/GDK-Tools/Source/Scripts/XGameSaveWrapper.cs

```diff
@@ -294,7 +294,7 @@ private void Update(string containerName, IDictionary<string, byte[]> blobsToSav
             }
 
             XGameSaveUpdateHandle updateHandle;
-            hr = SDK.XGameSaveCreateUpdate(containerHandle, null, out updateHandle);
+            hr = SDK.XGameSaveCreateUpdate(containerHandle, containerName, out updateHandle);
```

Comment:
intentional?

Reply:
ahh seems to be intentional due to the description

Reply:
Yes, it was intentional. Thank you for noticing and asking though!